packages/banner/src/react/index.tsx

```diff
@@ -66,43 +65,21 @@ const Banner = React.forwardRef((props, ref) => {
   )
 }) as BannerComponent
 
-Banner.displayName = 'Banner'
-
-interface AnchorProps extends HTMLPropsFor<'a'> {
-  href: string
-}
-interface ButtonProps extends HTMLPropsFor<'button'> {
-  href?: undefined
-}
-type ButtonComponent = React.ForwardRefExoticComponent<unknown> & {
-  (props: AnchorProps, ref?: RefFor<'a'>): JSX.Element
-  (props: ButtonProps, ref?: RefFor<'button'>): JSX.Element
-}
 const Button = React.forwardRef<
   HTMLAnchorElement | HTMLButtonElement,
-  AnchorProps | ButtonProps
+  React.ComponentProps<typeof DSButton>
```

Comment:
Yeah I recommend against it but this was one of those exceptions because it's wrapping a forwardRef Component the PSDS Button. Wouldn't of been so bad if you could import button without the forwardRef wrapping it.